internal/crypto/src/tests/raw_signature/signers.rs

```diff
@@ -19,8 +19,8 @@ use crate::raw_signature::{
 };
 
 #[test]
-// #[cfg_attr(all(target_arch = "wasm32", not(target_os = "wasi")), wasm_bindgen_test)]
-#[cfg(not(target_arch = "wasm32"))]
+// #[cfg_attr(all(target_arch = "wasm32", not(target_os = "wasi")),
```

Comment:
did you mean to leave all these in but commented out? Is there a reason to keep them?

Reply:
I left it as is but `cargo fmt` changed it. I'm not sure why they are here. 

I did remove the exclusion of the tests from WASI, since they run just fine.